velox/exec/Operator.cpp

```diff
@@ -415,6 +415,13 @@ void Operator::MemoryReclaimer::enterArbitration() {
   // The driver must be alive as the operator is still under memory arbitration
   // processing.
   VELOX_CHECK_NOT_NULL(driver);
+  if (FOLLY_UNLIKELY(!driver->state().isOnThread())) {
+    // NOTE: not all the memory arbitration are triggered from driver execution
```

Comment:
The wording here is confusing. It reads as "table scan prefetch" is an example of "driver executor" when it is the opposite.